application/controllers/exercises.py

```diff
@@ -38,6 +40,16 @@
     help='category parameter is required'
 )
 
+class RandExercises(Resource):
+    def get(self):
+        category = request.json['category']
+        duration = request.json['duration']
+
+        exercise = Exercise.query.filter(Exercise.category == category, Exercise.duration == duration)\
+        .order_by(func.random()).first()
```

Comment:
Now this looks like something I recognize!! Yay!

Reply:
Thanks for implementing the random functionality on the back-end! Makes it such a breeze for us